opensmtpd/handlers/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: set mta to opensmtpd
```

Comment:
Could Ansible alternatives module be used in here?
http://docs.ansible.com/ansible/latest/alternatives_module.html